peercrawler.py

```diff
@@ -151,6 +159,11 @@ def handle_incoming(connection: socket.socket, address, ctx: dict) -> Optional[t
                 logger.debug(f"Received peers from {address}")
                 incoming_peer_peers = keepalive.peers
 
+            elif header.msg_type == message_type(message_type_enum.confirm_ack):
+                confirm_response = confirm_ack.parse(header, payload)
+                if confirm_request.is_response(confirm_response):
+                    is_voting = True
```

Comment:
@dsiganos do you think it would be too excessive to wait 60 seconds for a confirm_ack packet? I have thought about reducing the timeout specifically for this kind of packets, since unlike the a keepalive of telemetry_ack, it will be usual that it is not received.

Reply:
It is a touch one, we want to give it every chance but we don;t want to wait too long for something that may never arrive.
I agree that one minute sounds too long. I'd go for 15 seconds.